modules/template/models/Template.php

```diff
@@ -125,6 +125,31 @@ public function validType($attribute, $model)
         }
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function afterSave($insert, $changedAttributes)
+    {
+        parent::afterSave($insert, $changedAttributes);
+
+        if (isset($changedAttributes['name'])) {
+            // Update to new template name when it is used as Allowed Template in Container Element
+            $definitions = ContainerDefinition::find()
+                ->where(['LIKE', 'dyn_attributes', '"templates":['])
+                ->andWhere(['LIKE', 'dyn_attributes', '"' . $changedAttributes['name'] . '"']);
```

Comment:
@yurabakhtin  Do we have to use the %template% name here or is the `%` not necessary?

Reply:
@luke- Yes, you are right, but I have tested and Yii adds the `%` for `LIKE` condition automatically, so we don't need to do it, otherwise we get them twice and it doesn't work as expected.